doc/release-notes-16378.md

```diff
@@ -1,5 +1,6 @@
 RPC
 ---
 - A new `send` RPC with similar syntax to `walletcreatefundedpsbt`, including
-  support for coin selection and a custom fee rate. Using the new `send` method
-  is encouraged: `sendmany` and `sendtoaddress` may be deprecated in a future release.
+  support for coin selection and a custom fee rate. The `send` RPC is experimental
```

Comment:
`Using it is encouraged once it's no longer experimental`: This sentence seems unnecessary, and somewhat confusing. There's no reason not to use it (i.e it being unsafe) as long as you know the interface might change in the future, and that is made clear here and in the help output. Will no-doubt be changed in the wiki pre-release.